src/main/java/com/github/krzysiekjodlowski/ox/BoardSizeOutOfBoundsException.java

```diff
@@ -0,0 +1,13 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * @author Krzysztof Jodlowski
+ */
+public class BoardSizeOutOfBoundsException extends Exception {
```

Comment:
InvalidBoardSize instead of BoardSizeOutOfBoundsException?

Reply:
Exception removed:)